src/ocp_postprocess/cluster_domain_rename/rename_utils.rs

```diff
@@ -844,3 +851,93 @@ pub(crate) fn fix_kapi_startup_monitor_pod_yaml(pod_yaml: &str, original_hostnam
     let replacement = format!(r"--node-name={}", hostname);
     Ok(pod_yaml.replace(&pattern, &replacement))
 }
+
+pub(crate) fn override_machineconfig_source(machineconfig: &mut Value, new_source: &str, path: &str) -> Result<()> {
```

Comment:
not sure if this file is the best place though most reused functions are already here

Reply:
let's move this file to be directly under the ocp_postprocess module instead of under domain rename, it was not intentionally meant to be shared